apps/prairielearn/src/lib/assets.ts

```diff
@@ -179,7 +179,13 @@ export function applyMiddleware(app: express.Application) {
       immutable: !config.devMode,
     }),
   );
-  router.use('/elements/:cachebuster', elementFiles);
+  router.use(
+    '/elements/:cachebuster',
+    (await import('../pages/elementFiles/elementFiles.js')).default({
+      publicQuestionEndpoint: false,
+      static: true,
+    }),
```

Comment:
This seems unnecessary; why not import at the top-level scope like we had before? This isn't `server.js` where we want to avoid overloading the top of the file with imports.

Reply:
sure, done